app/src/androidTest/java/com/example/sharingang/SellItemFragmentTest.kt

```diff
@@ -38,10 +39,10 @@ class SellItemFragmentTest {
         buttonCreate.check(ViewAssertions.matches(ViewMatchers.withText("Create Item")))
         buttonCreate.perform(ViewActions.click())
 
-        onView(ViewMatchers.withText("Sell")).perform(ViewActions.click())
-        // Wait for RecyclerView to remove old button
-        Thread.sleep(1000)
-        onView(ViewMatchers.withId(R.id.item_list_view_sell_button))
-            .check(ViewAssertions.matches(ViewMatchers.withText("Resell")))
+        onView(ViewMatchers.withId(R.id.item_list_view_title)).perform(ViewActions.click())
+
+        onView(ViewMatchers.withId(R.id.menuSell)).perform(ViewActions.click())
+
+        onView(ViewMatchers.withId(R.id.menuResell)).check(matches(ViewMatchers.isDisplayed()))
```

Comment:
We should probably change this code to either always use qualified imports (e.g. `ViewAssertions.matches`) or to just import `ViewAssertions.*`, and use `matches`. (I'd prefer the latter, but it doesn't really matter; consistency is more important).